cmd/lncli/cmd_macaroon.go

```diff
@@ -195,7 +197,7 @@ func bakeMacaroon(ctx *cli.Context) error {
 var listMacaroonIDsCommand = cli.Command{
 	Name:     "listmacaroonids",
 	Category: "Macaroons",
-	Usage:    "List all macaroons root key IDs in use",
+	Usage:    "List all macaroons root key IDs in use.",
```

Comment:
nit: change should be in previous commit, together with the punctuation of `bakemacaroon`.

Reply:
my bad, fixed